src/gemdat/trajectory.py

```diff
@@ -9,7 +9,7 @@
 import xml.etree.ElementTree as ET
 from itertools import compress
 from pathlib import Path
-from typing import Collection, Optional
+from typing import Collection, List, Optional
```

Comment:
You can use the built-in list type since 3.9.

```suggestion
from typing import Collection, Optional
```